scheduler/clients/javascript/tests/integration/integration.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { Calendar, INettuClient, User, CalendarEvent } from "../../lib";
+import { setupAccount } from "../helpers/fixtures";
+
+// This test suite is testing the specifications for our use cases
```

Comment:
Is the idea to do something similar to the `id-platform` `requirements.spec.ts`?

We should standardize on the name if so.

I would also make your test descriptions higher level than these ones, you can see in [requirements.spec.ts](https://github.com/meetsmore/meetsmore/pull/27088/files#diff-d4e6f0a19d74f4520424439bd10806a066241a24857abe1e6fe5e3040fb5d9d3) that I use `describe` for the actual requirement, and `it` for the test cases which fulfill it (if there are more than 1).

Requirements examples are:

- A user can create an account by logging in with Google.
- Users are notified when their calendar is updated.
- A user can delete their account.

Requirements define the desired functionality and less so the weedy details (e.g. it's not indicated above _how_ a user is notified, just that they are notified, then the test would be specific to email).

Reply:
It's indeed the goal, and I agree with you that I should likely use the same name + make the improvements you mentioned. I'll update the PR accordingly

Reply:
To be honest writing requirements is a bit of an _art_ rather than a science and it's also fairly subjective so :shrug:

Reply:
In the end, I've adapted with a mix of both. I can also adapt the `it()` more, but I admit I like the concepts of having the "describe" for the "higher level view" and the "it" for the "lower level view"